package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@adobe/node-httptransfer",
```

Comment:
didn't we agree it would be repetitive to have `node` in the name for the npm module? Switch to just `@adobe/httptransfer`?

The reason for `node` in `@adobe/node-fetch-retry` is because it is based off `node-fetch`
cc/ @alexkli

Reply:
Seems you're answering your question? (I don't understand where you're going)

Reply:
I mean, I would keep the node-* in there?
BEcause: < < The reason for node in @adobe/node-fetch-retry is because it is based off node-fetch > >

Reply:
The npm module name here should be `@adobe/httptransfer`.

The rule is:
- repo names of general purpose node modules that we open source are prefixed with `node-` so we avoid conflicts with other (possibly non-node) projects in the github.com/adobe org. this is the one and only reason we had these `node-` prefixes
- this is not needed for npm modules. having them prefixed with `node-` is normally repetitive (these are all node modules)
- for the `asset-compute-*` projects we have a very specific and long name already, no need for `node-` prefix in their repo names

And `node-fetch-retry` is a very special case as it started as wrapper around `node-fetch`, so we kept that.

Reply:
Name changed to httptransfer.